packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -147,10 +150,12 @@ export function DocSearchModal({
             searchSuggestions: [],
           },
         },
+        insights,
         navigator,
         onStateChange(props) {
           setState(props.state);
         },
+        // @ts-expect-error
```

Comment:
It's dangerous to ignore typing errors for such a foundational function. What error are you expecting here?

Reply:
@francoischalifour sorry this is still WIP so this will go away soon 😅

Reply:
This has been updated and removed now :)